tests/unit/common.py

```diff
@@ -248,6 +248,11 @@ def _launch_procs(self, num_procs):
                 f"Skipping test because not enough GPUs are available: {num_procs} required, {get_accelerator().device_count()} available"
             )
 
+        if get_accelerator().device_name() == 'xpu':
+            print("use non_daemonic_procs for xpu")
```

Comment:
Hi @ys950902 - could you remove the print, since the code below is quite readable?  And I tested this and it works, but could you let us know why this needs to be set or is recommended to be set?

Reply:
Hi @loadams, I have removed the print, yes, because some issues for ipex when using daemonic_proc, this can be a WA for XPU device to cover more unit test for multi-processing scenarios.